components/PlayerForm.js

```diff
@@ -221,6 +238,14 @@ export default function PlayerForm({ onSubmit, formName, defaultData }) {
         name="longitude"
         value={addressInfos.longitude ? addressInfos.longitude : defaultData?.longitude}
       />
+      {user && (
+        <input
+          type="hidden"
+          name="user"
+          value={defaultData?.user ? defaultData?.user : user[0]._id}
```

Comment:
I've not used this type before, but [mdn](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/hidden) tells me that the data is still visible in the DevTools. Is this data safe to be visible there?